index.js

```diff
@@ -0,0 +1,166 @@
+/* Ethical ad publisher JavaScript client
+ *
+ * Loads placement from Ethical Ad decision API. Searches for elements with
+ * `ethical-ad` data binding attributes and uses these attributes to query the
+ * decision API.
+ *
+ * This is native JavaScript, no JQuery. It uses the API JSONP interface to get
+ * around CORS and related issues. A script is added with a callback on
+ * `window`. The promise is rejected if there are errors with the request or the
+ * response doesn't look correct.
+ *
+ * Currently, only two parameters are supported with the ad placement: publisher
+ * id and the place type. All of this is determined by the server and this
+ * client so far only renders the API return HTML.
+ *
+ * This can be loaded async. CSS styles are preloaded via webpack
+ * `to-string-loader`.
```

Comment:
I'm ok with this for now. We may need to modify this or make it an option in the future. For tech focused sites, I'm sure some implement CSP and may block inline styles.

Reply:
Yeah true. Webpack has some capability here, but I'm not super familiar with it.